about.html

```diff
@@ -0,0 +1,94 @@
+            <hr class="conference-hr">
+            <div class="summit-intro">
+                <p class="mob-intro">The global summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and individuals.</p>
+                <p class="desktop-intro">The global summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and individuals who want to work together with us on shared projects that advance the cause of the Commons, free culture and open knowledge. So if you are active and engaged in the worlds of open content and knowledge - free software advocates, Wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together.</p>
+                <img src="./images/colored-circle.png">
+            </div>
+            <div class="plz-contact">
+                <p>Please contact us per Email for any further questions about CC Global Summit 2015!<br><strong>ccsummit2015@cckorea.org</strong></p>
+            </div>
+        </header>
+
+        <main>
+            <div class="summit-logo">
+                <h2 class="mobile-creative">Creative Commons<br>Global Summit 2015 Logo</h2>
+                <h2 class="desktop-creative">Creative Commons Global Summit 2015 Logo</h2>
+                <hr>
+                <p>The logo of CC Global Summit 2015 was decided through the logo competition from 8 June to 7 July.</p>
+                <p class="summit-det">After online public voting and Selection Committee's judging process, the logo from Naresh Agarwal won the prize. After consultation with Mr. Agarwal we optimized the prize-winning-logo for this year's summit as following :</p>
+                <img src="./images/summit-icon.jpg">
+            </div>
+            
+            <hr class="separator">
+
+            <div class="past-summit">
+                <h2>See the past CC Global Summit</h2>
+                <hr>
+                <p>Take a look at the last two CC Global Summits which took place in Buenos Aires and in Warsaw.</p>
+                <div class="past-summit-flex">
+                    <div><img src="./images/summit-2013.jpg"></div>
+                    <div><img src="./images/summit-2011.jpg"></div>
+                </div>
+            </div>
+            <div class="speaker-desktop extra-div">
+            </div>
+        </main>
+        <footer>
+            <div class="partners"><img src="./images/partners.jpg"></div>
+            <div class="partners-desktop"><img src="./images/about-footer.jpg"></div>
+        </footer>
```

Comment:
- [ ] As I said earlier, kindly make sure to create the footer section with code instead of screenshots of the behance design to make your project professional :recycle:

Reply:
Created all the footers with coding as per instructions